exporters/etw/include/opentelemetry/exporters/etw/etw_tracer.h

```diff
@@ -791,6 +881,27 @@ class Span : public trace::Span
   }
 
 public:
+  /**
+   * @brief Update Properties object with current Span status
+   * @param evt
+   */
+  void UpdateStatus(Properties &evt)
+  {
+    /* Should we avoid populating this extra field if status is unset? */
+    if ((status_code_ == trace::StatusCode::kUnset) || (status_code_ == trace::StatusCode::kOk))
+    {
```

Comment:
Why report `kUnset` as success, for other exporter like [Zipkin](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/zipkin.md#status), the status is not reported for `kUnset`.

Reply:
Okay, I will change that. I was not sure.